src/rgw/rgw_op.cc

```diff
@@ -6189,6 +6194,11 @@ void RGWInitMultipart::execute(optional_yield y)
     upload_id = upload->get_upload_id();
   }
   s->trace->SetAttribute(tracing::rgw::UPLOAD_ID, upload_id);
+
+  // set an attribute called "upload_id", 
+  // initialize it with value 0 (implement the algo later)
+  s->trace->SetAttribute(tracing::rgw::UPLOAD_CHECKSUM, 0);
+
```

Comment:
have you implemented the algo?

Reply:
No, I haven't. 
I found that there already exists similar attribute like MD5 check in Jaeger.

Reply:
ok.
please remove the irrelevant files from the PR. you need to add only `rgw_op.cc` and `rgw_tracer.h`, so the other files that may be generated during the build are not supposed to be part of the PR

Reply:
We should really avoid using MD5 for this sort of thing.  The obvious choice would be SHA256 or maybe SHA512 (/2), whichever is faster.  An apparently fast alternative might be Blake2b.